app-header/test/app-header.html

```diff
@@ -107,18 +106,21 @@
 
     function assertHeaderIsFullSized(header) {
       var headerClientRect = header.getBoundingClientRect();
-      assert.equal(headerClientRect.top, 0);
-      assert.equal(headerClientRect.bottom, headerClientRect.height);
+      header._updateScrollState(header._scrollTop, true);
+      assert.equal(headerClientRect.top, 0, 'full sized');
+      assert.equal(headerClientRect.bottom, headerClientRect.height, 'full sized');
     }
 
     function assertHeaderIsCondensed(header) {
       var headerClientRect = header.getBoundingClientRect();
-      assert.equal(headerClientRect.top + headerClientRect.height, header._primaryEl.offsetHeight);
+      header._updateScrollState(header._scrollTop, true)
```

Comment:
Missing `;`

Also, was this the reason why app-header tests were failing in Safari 8? Seems to pass on my VM and may just be flaky (which if this fixes would be great).